docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,137 @@
+
+Create the Google Cloud account and start with `gcloud`:
+
+https://cloud.google.com/pubsub/docs/quickstart-cli
+
+Login to Google Cloud
+
+.. code-block:: bash
+
+   $ cd /tmp/
+   $ gcloud init
+
+Create new project in GCE
+
+.. code-block:: bash
+
+   $ GCE_PROJECT_NAME="My Test Project"
+   $ GCE_PROJECT_ID="my-test-project-`date +%F`"
+   $ gcloud projects create "$GCE_PROJECT_ID" --name "$GCE_PROJECT_NAME"
+   Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/my-test-project-2018-05-11].
+   Waiting for [operations/cp.7874714971502871878] to finish...done.
+
+Set new project as default
+
+.. code-block:: bash
+
+   $ gcloud config set project $GCE_PROJECT_ID
+
+Enable Billing for the project
+
+   $ GCE_ACCOUNT_ID=$(gcloud beta billing accounts list --filter="My Billing Account" --format="value(ACCOUNT_ID)")
+   $ gcloud billing projects link "$GCE_PROJECT_ID" --billing-account="$GCE_ACCOUNT_ID"
+
+Create Service Account and assign it proper rights
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_DISPLAY_NAME="My_Service_Account"
+   $ GCE_SERVICE_ACCOUNT_NAME="my-service-account"
+   $ gcloud iam service-accounts create "$GCE_SERVICE_ACCOUNT_NAME" --display-name "$GCE_SERVICE_ACCOUNT_DISPLAY_NAME"
+
+Generate keys for new service account
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list --filter="$GCE_SERVICE_ACCOUNT_NAME" --format="value(email)")
+   $ gcloud iam service-accounts keys create --iam-account $GCE_SERVICE_ACCOUNT_EMAIL key.json
+
+Assign role to the serviceaccount and bind it to the project
+
+.. code-block:: bash
+
+   $ GCE_EMAIL="petr.ruzicka@gmail.com"
+   $ gcloud iam service-accounts add-iam-policy-binding "$GCE_SERVICE_ACCOUNT_EMAIL" --member="user:$GCE_EMAIL" --role="roles/owner"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/container.clusterAdmin"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/iam.serviceAccountActor"
+
+Start Kqueen (if necessary) and obtain bearer token
+
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+Create new organization "testorganization" with new user/password "testusername / testpassword":
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data '{ "name": "testorganization", "namespace": "testorganization" }' -X POST 127.0.0.1:5000/api/v1/organizations | jq
```

Comment:
$

Reply:
I'm not going to change it.
According the https://docs.openstack.org/doc-contrib-guide/index.html this is the correct way.